### PR TITLE
Follow-up for native support for Sparse Hamiltonians expval()

### PR DESCRIPTION
**Pull request by AmintorDusko**
### Before submitting

Please complete the following checklist when submitting a PR:

- [x] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [x] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [x] Ensure that the test suite passes, by running `make test`.

- [x] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [x] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**
Small follow-up PR for fixing problems with wheels.
**Description of the Change:**

**Benefits:**
Properly render Readme on PyPI
**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -236,12 +236,15 @@ The PennyLane lightning plugin is **free** and **open source**, released under
...
 Acknowledgements
 ================
 
 PennyLane Lightning makes use of the following libraries and tools, which are under their own respective licenses:
 
-- **pybind11:** https://pybind11.readthedocs.io/en/stable/ [`License <https://github.com/pybind/pybind11/blob/master/LICENSE>`_]
-- **Kokkos Core:** https://github.com/kokkos/kokkos [`License <https://github.com/kokkos/kokkos/blob/master/LICENSE>`_]
-- **Kokkos Kernels:** https://github.com/kokkos/kokkos-kernels [`License <https://github.com/kokkos/kokkos-kernels/blob/master/LICENSE>`_]
\ No newline at end of file
+- **pybind11:** https://pybind11.readthedocs.io/en/stable/
```
Comment (maliasadi) [inline README.rst:246]:
```suggestion
- **pybind11:** https://github.com/pybind/pybind11
```

**Review by maliasadi — APPROVED** [review]
Thank you @AmintorDusko. Just a minor suggestion; no blocking and happy to approve anyway.

File: README.rst (hunk 1 above)
Comment (mlxd) [reply README.rst:246]: Do you think it better to use Github here rather than RTD?

Comment (maliasadi) [reply README.rst:246]: Comparing to provided links for kokkos and kokkos-kernels, I think the github link is a better call. But, I'm :100: okay with any decisions for this regard.

Comment (AmintorDusko) [reply README.rst:246]: Thank you, @maliasadi!

**Comment by maliasadi** [comment]
Hi @AmintorDusko. Great job on both PRs :rocket: I also have some general suggestions for your other PR #283 (will be added shortly). Could you please apply those in this PR before merging?

**Pull request merged**